molo/commenting/templates/comments/comment.html

```diff
@@ -37,7 +35,12 @@ <h6 class="comment__description">
         </a>
       {% endif %}
       {% if request.user.is_authenticated%}
-        <a href="{% url 'molo.commenting:molo-comments-reply' node.pk %}#comment-form" class="call-to-action__nav-item call-to-action__nav-item--response">{% trans "Reply"%}</a>
+        <div class="comment__reply-meta">
+          <a href="{% url 'molo.commenting:molo-comments-reply' node.pk %}#comment-form" class="call-to-action__nav-item call-to-action__nav-item--response comment__reply-meta-link">{% trans "Reply"%}</a>
+          <a href="{% url 'molo.commenting:molo-comments-reply' node.pk %}#comment-form" class="comment__reply-meta-link comment__reply-meta-replies-list">
+            {% trans "Replies"%}
```

Comment:
The counter seems to be missing here - see how it was done previously: https://github.com/praekelt/molo.commenting/pull/42/files#diff-4f03d5f24d520b64a1d3e834956f0d05R38